backends/advanced/src/advanced_omi_backend/controllers/system_controller.py

```
@@ -272,171 +247,100 @@ async def get_speaker_service_status():
-            
-            return {
-                "message": "Configuration is valid",
-                "status": "success"
-            }
-            
-        except ValueError as e:
-            return JSONResponse(
-                status_code=400, content={"error": f"Configuration validation failed: {str(e)}"}
-            )
-        
+            raise ValueError(f"Invalid YAML syntax: {str(e)}")
```

Ruff (0.14.8)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)